Build.cs

```diff
@@ -412,6 +417,7 @@ private bool ValidateROM(string FileName)
 
         private void MakeROM(string InFile, string FileName, BackgroundWorker worker)
         {
+            if(!cLogOnly.Checked)
```

Comment:
There shouldn't be a "Spoiler Log Only" checkbox. Just a checkbox for each type of output. N64, VC, SpoilerLog, SpoilerLog (HTML)